src/auxiliary/special_elixirs.jl

```diff
@@ -36,6 +36,16 @@ julia> redirect_stdout(devnull) do
 ```
 """
 function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
+    # Check that all kwargs exist as assignments
+    code = read(elixir, String)
+    expr = Meta.parse("begin \n$code \nend")
+    expr = insert_maxiters(expr)
```

Comment:
`insert_maxiters` is also called below. I think it should be called only once

Reply:
Here, I call it with the full expression that I get from wrapping the elixir in `begin end`. Below, it is passed to `Base.include`, where it is called for every smaller expression.